apps/fabric-website/scripts/createFlightConfig.js

```diff
@@ -0,0 +1,63 @@
+/*
+  Creates a configuration JS file that exposes the build ID and base URL that the site should load.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { argv, logger } = require('@uifabric/build/just-task').just;
+
+module.exports = function createFlightConfigTask() {
+  return function createFlightConfigTask() {
+    let date = new Date();
+    let today = date.getFullYear() + ('0' + (date.getMonth() + 1)).slice(-2) + ('0' + date.getDate()).slice(-2);
```

Comment:
Is there not some date formatting method we can use for this?

Reply:
sigh, this is code from Peter.. probably true, but not really interested in rewriting it all the way right now